go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/crossplane/function-sdk-go
 
 go 1.23
 
-toolchain go1.23.1
+toolchain go1.23.2
```

Comment:
Should Renovate also find and update the go version in https://github.com/crossplane/function-sdk-go/blob/main/.github/workflows/ci.yml#L13?

Reply:
aha voila, it was the next PR and it was rate limited - i forced its creation now: https://github.com/crossplane/function-sdk-go/pull/192